packages/react-dom/src/client/ReactDOM.js

```diff
@@ -45,6 +45,7 @@ import {
   DOCUMENT_FRAGMENT_NODE,
 } from '../shared/HTMLNodeType';
 import {ROOT_ATTRIBUTE_NAME} from '../shared/DOMProperty';
+import {REACT_PORTAL_TYPE} from 'shared/ReactSymbols';
```

Comment:
@brandonvilla21
Isn't that supposed to be like that:  ` '../shared/ReactSymbols';`

Reply:
@PatrykDeja 
I'm accessing to a more global `shared` folder in that way. I suppose the project is using a config to set an alias in order to access to that global shared folder located in `packages/shared`. Otherwise, if I use `'../shared'` I'll be accessing to the parent shared folder where is not the file that I want to reach